### ISSUE TITLE
Add support for websocket_tail output

### ISSUE THREAD

**Issue by muhlpachr** [Jeffail/benthos#296]
Would you consider to add WebSocket output with functionality alike https://github.com/allinurl/gwsocket (allowing message trashing in case there is not any client) to be able to watch/check stream output just from web browser please ?

**Comment by Jeffail** [comment]
Hey @muhlpachr, if there are multiple clients connected would you expect messages to be broadcast to all of them?

There's a possibility of adding an output mechanism to wrap `http_server`, where if it doesn't accept a transaction the message is dropped, which would get you partially there. However, I'd need to implement a different mechanism for publishing messages to N clients.

**Comment by muhlpachr** [comment]
@Jeffail yes, if there are multiple clients connected, I would like to broadcast message to all of them like https://github.com/allinurl/gwsocket.

It would be nice to be able to do that without additonal component like gwsocket.
I do understand, that adding (publishing over http server) client JavaScript terminal together with WebSocket is not usual stream processing output. I consider it useful for troubleshooting and debugging.

Streaming through HTTP GET is limited by web browser timeouts for GET and it also do not reconnect autonomously from web browser in case of connectivity troubles.

What kind of wrap for `http_server` do you mean please ?

**Comment by twmb** [comment]
Closing as part of a backlog cleanup. This has been open since 2019 with no further activity. For debugging/inspecting pipeline output, there are other approaches available today. If there's renewed interest, feel free to open a new issue.
